src/modules/whats-new/components/whats-new-modal.tsx

```diff
@@ -68,31 +72,40 @@ export default function WhatsNewModal( { showModal, onClose }: WhatsNewModalProp
 					code: <code />,
 				},
 			} ),
-			learnMoreUrl: 'https://developer.wordpress.com/docs/developer-tools/studio/cli/',
+			learnMoreUrl: getDocsLink( locale, 'cli' ),
 		},
 		{
 			image: customDomainIllustration,
 			title: __( 'Choose a custom domain with HTTPS support' ),
 			description: __(
 				'Easily identify your local Studio sites with custom domain names. Personalize and organize your workflow!'
 			),
-			learnMoreUrl: 'https://wordpress.com/blog/2025/03/31/studio-custom-domains-https/',
+			learnMoreUrl: translateLink(
```

Comment:
What if we followed the same approach as we took for docs, so we would have `getBlogLink` that checks if there is an internationalized URL and returns it?

Reply:
I thought about that. I'll create a second PR for that.

Reply:
Wouldn't it be mainly about renaming `src/lib/translate-link.ts` and its methods?

Reply:
I miss-read your first comment. I thought you were suggesting to unify both functions, because I was thinking on that from the moment I created this PR. I also like having all the links indexed.

Check out this PR, it has the best from both worlds: https://github.com/Automattic/studio/pull/1375